dotCMS/src/main/webapp/dotcms-webcomponents/p-op8tbxkh.system.entry.js

```diff
@@ -0,0 +1 @@
.name="";this.label="";this.hint="";this.options="";this.required=false;this.disabled=false;this.requiredMessage="This field is required";this.value="";this.dotValueChange=i(this,"dotValueChange",7);this.dotStatusChange=i(this,"dotStatusChange",7)}t.prototype.componentWillLoad=function(){this.value=this.value||"";this.validateProps();this.emitValueChange();this.status=s(this.isValid());this.emitStatusChange()};t.prototype.componentDidLoad=function(){var t=this;var e=["dottype"];var i=this.el.querySelectorAll('input[type="checkbox"]');setTimeout((function(){var n=v(Array.from(t.el.attributes),e);i.forEach((function(t){y(t,n)}))}),0)};t.prototype.optionsWatch=function(){var t=o(this,"options");this._options=u(t)};t.prototype.valueWatch=function(){this.value=this.value||""};t.prototype.reset=function(){return __awaiter(this,void 0,void 0,(function(){return __generator(this,(function(t){this.value="";this.status=s(this.isValid());this.emitValueChange();this.emitStatusChange();return[2]}))}))};t.prototype.render=function(){var t=this;var e=l(this.status,this.isValid(),this.required);return n(a,{class:Object.assign({},e)},n("dot-label",{label:this.label,required:this.required,name:this.name},n("div",{"aria-describedby":d(this.hint),tabIndex:this.hint?0:null,class:"dot-checkbox__items"},this._options.map((function(e){var i=e.value.trim();return n("label",null,n("input",{class:f(t.isValid()),name:p(t.name),type:"checkbox",disabled:t.disabled||null,checked:t.value.indexOf(i)>=0||null,onInput:function(e){return t.setValue(e)},value:i}),e.label)})))),c(this.hint),h(!this.isValid(),this.requiredMessage))};t.prototype.validateProps=function(){this.optionsWatch()};t.prototype.setValue=function(t){this.value=this.getValueFromCheckInputs(t.target.value.trim(),t.target.checked);this.status=b(this.status,{dotTouched:true,dotPristine:false,dotValid:this.isValid()});this.emitValueChange();this.emitStatusChange()};t.prototype.getValueFromCheckInputs=function(t,e){var i=this.value.trim().length?this.value.split(","):[];var n=new Set(i);if(e){n.add(t)}else{n.delete(t)}return Array.from(n).join(",")};t.prototype.emitStatusChange=function(){this.dotStatusChange.emit({name:this.name,status:this.status})};t.prototype.isValid=function(){return this.required?!!this.value:true};t.prototype.emitValueChange=function(){this.dotValueChange.emit({name:this.name,value:this.value})};Object.defineProperty(t.prototype,"el",{get:function(){return r(this)},enumerable:true,configurable:true});Object.defineProperty(t,"watchers",{get:function(){return{options:["optionsWatch"],value:["valueWatch"]}},enumerable:true,configurable:true});Object.defineProperty(t,"style",{get:function(){return".dot-checkbox__items{display:-ms-flexbox;display:flex;-ms-flex-direction:column;flex-direction:column}.dot-checkbox__items label{display:-ms-flexbox;display:flex;-ms-flex-align:center;align-items:center}.dot-checkbox__items input{margin:0 .25rem 0 0}"},enumerable:true,configurable:true});return t}())}}}));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A 'return', 'break', 'continue', or 'throw' statement should be the last in a block.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5062311)